tests/Feature/PageArchiveTest.php

```diff
@@ -43,4 +43,43 @@ public function it_orders_streams_from_latest_to_oldest(): void
                 'Finished three days ago',
             ]);
     }
+
+    /** @test */
+    public function we_can_search_for_streams_on_title(): void
```

Comment:
Please use `it` like `it_searches_...` for the method names to keep it consistent with other tests.